### PR TITLE
debugability: performance counters

### PR DESCRIPTION
**Pull request by mwasko**
Performance counters usage descriptions added

### CODE REVIEW

**Comment by lgirdwood** [comment]
@anton-intel fyi.

File: developer_guides/debugability/perf-counters/index.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Performance Counters usage
+**************************
+
+Currently performance counters can be only enabled statically during FW build in
+one of two ways:
+
+#. Select "Performance counter" from "Debug" menu using make menuconfig
+
+#. Add "CONFIG_PERFORMANCE_COUNTERS=y" to specific FW config (e.g. tigerlake_defconfig)
```
Comment (anton-intel) [inline developer_guides/debugability/perf-counters/index.rst:25]: Currently, you can only enable performance counters statically during FW build in
one of two ways:

- Select **Performance counter** from the **Debug** menu using ``make menuconfig``

- Add ``CONFIG_PERFORMANCE_COUNTERS=y`` to a specific FW configuration, for example, tigerlake_defconfig.

File: developer_guides/debugability/perf-counters/index.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+Once the the performance counters are enabled they are logged periodically for
+each active component with the pipeline period frequency.
```
Comment (anton-intel) [inline developer_guides/debugability/perf-counters/index.rst:29]: After you enable the performance counters, they are logged periodically for
each active component with the pipeline period frequency.

File: developer_guides/debugability/perf-counters/index.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+   .. code-block:: bash
+
+      MCPS = cpu_ticks / (pipeline_period[s] * 10^6)
+      
+      // for common pipeline_period = 1ms it can be simplified to
+      MCPS = cpu_ticks / 1000
+
+In the trace example above the cpu_ticks = 8136, the pipeline_period is 1ms so
+the demux consumption equals 8,136 MCPS
```
Comment (anton-intel) [inline developer_guides/debugability/perf-counters/index.rst:60]: In the trace example above, *cpu_ticks* is 8136, *pipeline_period* is 1 ms, so
the demux consumption equals 8,136 MCPS.

**Comment by lgirdwood** [comment]
@anton-intel good for you now ?

File: developer_guides/debugability/perf-counters/index.rst (hunk 1 above)
Comment (anton-intel) [reply developer_guides/debugability/perf-counters/index.rst]: Can we make it a bullet list? A numbered list implies that both steps should be taken...

**Comment by anton-intel** [comment]
> @anton-intel good for you now ?

@lgirdwood Just a minor comment about the bullet list.

**Review by anton-intel — APPROVED** [review]

**Pull request merged**